src/ProcessHosts/Elastic.ProcessHosts/Elasticsearch/Process/ElasticsearchProcess.cs

```diff
@@ -191,9 +191,6 @@ public override void Start()
 			if (!this.FileSystem.Directory.Exists(this.PrivateTempDirectory))
 				this.FileSystem.Directory.CreateDirectory(this.PrivateTempDirectory);
 
-			if (!this.FileSystem.Directory.Exists(this.GCLogsDirectory))
```

Comment:
Can delete the `GCLogsDirectory` property and setter.

Reply:
Thanks, will fix